tests/tests_placebo.py

```diff
@@ -132,6 +132,28 @@ def test_create_lambda_function_local(self, session):
             function_name='test_lmbda_function55',
         )
 
+    @placebo_session
+    def test_create_lambda_function_docker(self, session):
+        bucket_name = 'lmbda'
+        docker_image_uri = 'docker_image_uri'
+
+        z = Zappa(session)
+        z.aws_region = 'us-east-1'
+        z.load_credentials(session)
+        z.credentials_arn = 'arn:aws:iam::12345:role/ZappaLambdaExecution'
+
+        arn = z.create_lambda_function(
+            bucket=bucket_name,
+            docker_image_uri=docker_image_uri,
+            function_name='test_lmbda_function55',
+        )
+
+        arn = z.update_lambda_function(
+            bucket=bucket_name,
+            docker_image_uri=docker_image_uri,
+            function_name='test_lmbda_function55',
+        )
```

Comment:
Coveralls is saying that `update_lambda_function ` is not covered by tests but it clearly is https://coveralls.io/builds/36035314/source?filename=zappa%2Fcore.py#L1110